### PR TITLE
Initial Hello World with Express

### PR DESCRIPTION
**Pull request by codydaig**

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+var gulp = require('gulp');
+var ts = require('gulp-typescript');
+var rimraf = require('gulp-rimraf');
+var nodemon = require('gulp-nodemon');
+ 
+gulp.task('cleanBuiltDir', function(){
```
Comment (trevorhanus) [inline gulpfile.js]: spaces: function () {

File: gulpfile.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+ 
+gulp.task('buildFrontEnd', ['cleanBuiltDir'],  function () {
+  var tsResult = gulp.src('src/**/*.ts')
+    .pipe(ts({
+        module: 'CommonJS'
+      }));
+  return tsResult.js.pipe(gulp.dest('build/'));
+});
+ 
+gulp.task('nodemon', ['buildBackEnd', 'buildFrontEnd', 'watch'], function(){
```
Comment (trevorhanus) [inline gulpfile.js]: spaces

**Comment by trevorhanus** [comment]
made a couple comments, really minor syntax stuff

**Comment by codydaig** [comment]
@trevorhanus Thanks!

**Pull request merged**